packages/expo-secure-store/src/SecureStore.ts

```diff
@@ -202,6 +202,14 @@ export function getItem(key: string, options: SecureStoreOptions = {}): string |
   return ExpoSecureStore.getValueWithKeySync(key, options);
 }
 
+/**
+ * Checks if the value can be saved with `requireAuthentication` option set to `true`.
```

Comment:
```suggestion
 * Checks if the value can be saved with `requireAuthentication` option enabled.
```

The last part of the sentence sounds confusing. We could either provide complete context that the "when that option is set to true" or just say "when that option is enabled" and be concise about it.